frontend/mulighetsrommet-cms/schemas/arrangor.js

```diff
@@ -0,0 +1,39 @@
+import { BiBuildings } from "react-icons/bi";
+
+export default {
+  name: "leverandor",
+  title: "Leverandor",
+  type: "document",
+  icon: BiBuildings,
+  fields: [
+    {
+      name: "title",
```

Comment:
Skal denne hete title? Eller burde den hete `name`? Altså `name: "name`?

Reply:
![image](https://user-images.githubusercontent.com/21334782/171400229-fcc03865-866f-4d9c-a2ca-d864b1a940ef.png)

Reply:
Etter litt graving fant vi ut av man kan bruke `preview`-feltet i schema for å definere hvilket felt som skal bli title. Les mer her https://www.sanity.io/docs/previews-list-views